doc/modules/ROOT/pages/common-datasets.adoc

```diff
@@ -92,6 +92,44 @@ The default is to load IMDB dataset as `undirected = True`. If loaded as directe
 |===
 
 
+=== LastFM
+
+A heterogeneous graph that is used to benchmark link prediction models, for example used by MAGNN: Metapath Aggregated Graph Neural Network for Heterogeneous Graph Embedding.
+The data and licensing is from https://grouplens.org/datasets/hetrec-2011/[HetRec 2011]
+The original raw data is from http://www.lastfm.com/[LastFM].
+
+The graph contains User and Artists as nodes, each with a `rawId` field that corresponds to the ids from the raw data.
+The `rawId` could be used, for example, to check the artist names by referring back to the HetRec'11 .dat files.
+
+There are three types of relationships.
+`IS_FRIEND` which represents User-User friendship.
+
+`LISTEN_TO` between User and Artist.
+It has a relationship property `weight` that represents the number of times the user listened to the artist.
+
+`TAGGED` between User and Artist.
+It has a `tagID` property where each `tagId` represents a genre,
+It also has `day`, `month`, `year` properties, as well as an equivalent `timestamp` property converted from the day-month-year,
+which represents when the user tagged that artist with that `tagId`.
+Note that a user can tag the same artist at different times with different `tagId`.
+
+The default is to load LastFM dataset as `undirected = True`.
```

Comment:
It is a little strange to consider the `TAGGED` and `LISTEN_TO` relationships as undirected. But in some sense the direction is encoded semantically with the intuition of the labels. I wonder if the model can understand that though...

Reply:
I agree, I think the main restriction is that our pipelines require undirected graphs.